src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -13,6 +13,7 @@ import StringConstant from '../../../utils/string/StringConstant';
 import useIsReady from '../../../hooks/useIsReady';
 import {COLORS} from '../../../utils/theme';
 import {Context} from '../../../context/Store';
+import {colors} from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `colors` is added but the old import statement for `COLORS` from `'../../../utils/theme'` is not removed. This could lead to confusion and potential bugs if both are used interchangeably in the code. Please ensure to remove the unused imports.
```diff
- import {COLORS} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->